k8s/commander.sh

```diff
@@ -31,6 +32,12 @@ function deploy {
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" rollouts.status-cron
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" deployments.create-web --apply --tag full-${COMMIT_HASH}
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" rollouts.status-web
+
+    # run post-deployment tasks
+    echo "Running post-deployment tasks"
+    SUMO_POD=$(${KUBECTL_BIN} -n "${K8S_NAMESPACE}" get pods | grep "${K8S_NAMESPACE}" | cut -d ' ' -f 1 | head -n 1)
```

Comment:
it might be easier to use the jsonpath output filter of kubectl to get the first pod:
```
kubectl -n sumo-stage get pods -o=jsonpath={.items[0].metadata.name}
```

See also: https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
Oooooo... nice! Thanks. I'll update. That is better. 🌮

Reply:
Updated 🎉

Reply:
:taco: :taco: :taco: